### PR TITLE
Add landing page to Tracing View (Ticket #850)

### PR DESCRIPTION
**Pull request by arne-z**
Upon opening the tracing view in the FCL App, users are currently greeted by an empty white canvas, which might make users think something isn't working, and does not clearly guide users to what their first steps should be.

The Goal of this PR is to replace this empty Canvas with some text or illustration that guides users first steps and presents a better first experience.

See Ticket #850 for details and mockups.

### CODE REVIEW

**Comment by arne-z** [comment]
The new look of the landing page, as discussed with @bfrgottscma. 

localhost_4200_tracing

**Review by ruegenbfr — APPROVED** [review]
Please see my remarks. They are merely suggestions.

File: src/app/tracing/components/main-tracing.component.html [hunk 1]
```diff
@@ -6,13 +6,37 @@
     >
         <mat-sidenav-content class="mat-sidenav-content-container">
             <div id="mainContent">
+                <div
```
[the remark this answers is not in the document]
Comment (arne-z) [reply src/app/tracing/components/main-tracing.component.html:9]: Done.

File: src/app/tracing/state/tracing.selectors.ts [hunk 2]
```diff
@@ -28,6 +28,11 @@ export const getTracingActive = createSelector(
     (state) => state.tracingActive,
 );
 
+export const getModelLoaded = createSelector(
```
[the remark this answers is not in the document]
Comment (arne-z) [reply src/app/tracing/state/tracing.selectors.ts]: Also done.

**Review by j-hha — APPROVED** [review]

**Pull request merged**